app/views/redirect/return_to_sp/cancel.html.erb

```diff
@@ -0,0 +1,42 @@
+<% self.title = t('titles.cancel_exit_login', app_name: APP_NAME) %>
+
+<%= render StatusPageComponent.new(status: :warning) do |c| %>
+  <% c.with_header { t('headings.cancellations.login_cancel_prompt', app_name: APP_NAME) } %>
+
+  <p>
+    <%= t('sign_up.login_cancel.header', app_name: APP_NAME) %>
+  </p>
+
+  <ul class="usa-list">
+    <li><%= t('sign_up.login_cancel.bullet1', app_name: APP_NAME, sp_name: decorated_sp_session.sp_name) %></li>
+    <li>
+      <%= t(
+            'sign_up.login_cancel.bullet2_html', app_name: APP_NAME,
+                                                 link_html: link_to(
+                                                   t('sign_up.login_cancel.account_page'),
+                                                   account_path,
+                                                 )
+          ) %>
+    </li>
+  </ul>
+
+  <div class="grid-row">
+    <div class="grid-col-6">
+      <%= link_to(
+            sign_up_completed_path,
+            method: :get,
+            class: 'usa-button usa-button--wide usa-button--big margin-bottom-2 margin-top-3',
+          ) do %>
+        <%= t('sign_up.login_cancel.keep_going') %>
+      <% end %>
+
+      <%= link_to(
+            @redirect_url,
+            class: 'usa-button usa-button--wide usa-button--outline usa-button--big',
+          ) do %>
+        <%= t('sign_up.login_cancel.exit', app_name: APP_NAME) %>
+      <% end %>
```

Comment:
`method: :get` is unnecessary for `link_to`, since it's the default. Also, we can use `ButtonComponent` here to deal with some of the implementation detail of classes. It was made a bit more usable after #10464 (may need to rebase).

```suggestion
      <%= render ButtonComponent.new(
            url: sign_up_completed_path,
            big: true,
            wide: true,
            class: 'margin-bottom-2 margin-top-3',
          ).with_content(t('sign_up.login_cancel.keep_going')) %>

      <%= render ButtonComponent.new(
            url: @redirect_url,
            big: true,
            wide: true,
            outline: true,
          ).with_content(t('sign_up.login_cancel.exit', app_name: APP_NAME)) %>
```